app/models/nic/interface.rb

```diff
@@ -69,6 +69,10 @@ def bridge
       attrs[:bridge]
     end
 
+    def nic_delay
+      return self.attrs['nic_delay'] if self.attrs['nic_delay'].present?
```

Comment:
Should this be delegated to the subnet?
I don't see anywhere where we set this attribute on the interface or use it?